python/HeaderService/hslib.py

```diff
@@ -343,11 +343,15 @@ def collect(self, keys):
     def collect_from_HeaderService(self):
         """Collect and update custom meta-data generated or transformed by the HeaderService"""
         self.DATE_OBS = hutils.get_date_utc(self.metadata['DATE-OBS'])
+        self.DATE_BEG = hutils.get_date_utc(self.metadata['DATE-BEG'])
```

Comment:
I'm pretty sure that @ktlim wants us to use TAI in all our headers (so TIMESYS="TAI").

Reply:
That's the plan, but we are not there yet.